src/misc/utils.ts

```diff
@@ -29,13 +29,13 @@ export function assert(
             let part: string;
 
             if (detail instanceof ASTNode) {
-                part = detail.type + " #" + detail.id;
```

Comment:
This if case can be removed (its already handled in pp()).

Reply:
Addressed in https://github.com/ConsenSys/solc-typed-ast/commit/6e612113efada422eb324ed77841541855af8c10